css/style.css

```diff
@@ -0,0 +1,490 @@
+* {
+  box-sizing: border-box;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background: #f4f5f6;
+  font-family: "Roboto", sans-serif;
+}
+
+.h2,
+.h3,
+.h4,
+.h5,
+.h6 {
+  line-height: 1em;
+}
+
+/* Navigation Styles */
```

Comment:
Please remove all the comments for your CSS and HTML files:
![image](https://user-images.githubusercontent.com/34653764/84458967-34258980-ac5e-11ea-93c0-21eae69801d5.png)

Reply:
The inline comment here is talking about commenting codes like(//console.log("Hello")) and not segmentations like <--Header navigation-->